src/components/infoTooltips/EModeTooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import { Trans } from '@lingui/macro';
+import { Link } from '../primitives/Link';
+
+import { TextWithTooltip, TextWithTooltipProps } from '../TextWithTooltip';
+
+export const EModeTooltip = ({ ...rest }: TextWithTooltipProps) => {
+  return (
+    <TextWithTooltip {...rest}>
+      <Trans>
+        E-Mode increases your LTV for a selected category of assets up to 97%.{' '}
```

Comment:
does it make sense that this is hardcoded 97%? On the asset details we should be able to just use the actual ltv from the asset in emode. Alternatively we could remove the "up to 97%".

Reply:
Added your suggestion, and using now emode ltv from reserve